docs/_data/jekyll_variables.yml

```diff
@@ -51,10 +51,10 @@ site:
       <code>name</code>, <code>basename</code> and <code>extname</code>.
   - name: site.html_pages
     description: >-
-      A subset of `site.pages` listing those which end in `.html`.
+      A subset of <code>site.pages</code> listing those which end in <code>.html</code>.
   - name: site.html_files
     description: >-
-      A subset of `site.static_files` listing those which end in `.html`.
+      A subset of <code> site.static_files</code> listing those which end in <code>.html</code.
```

Comment:
<img width="677" alt="variables-error" src="https://user-images.githubusercontent.com/103008/67688562-505c0b00-f99a-11e9-835e-c298c7fb7ff0.png">

This change introduce a rendering problem, see https://deploy-preview-7879--jekyllrb.netlify.com/docs/variables/

Reply:
I have missed one `>` character